text/0000-security-advisories.md

```diff
@@ -0,0 +1,259 @@
+- Feature Name: security_advisories
+- Start Date: 2016-08-24
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Crates.io should offer an API to release security advisories for crates. Cargo
+should offer a new command for the same purpose and warn about vulnerable crate
+versions during compilation.
+
+# Motivation
+[motivation]: #motivation
+
+When compared to other ecosystems such as Python's, Rust's broader community
+prefers many single-purpose crates over larger monoliths. This situation,
+together with the strongly encouraged practice of pinning MINOR versions of
+dependencies, slows down the propagation of critical security fixes.
```

Comment:
I don't understand what you mean. Dealing with upgrades is not concern of this RFC at all.

Also the new, completely rewritten example doesn't literally mention minor versions and avoids general statements about pinning.